blue-common/src/main/resources/application.conf

```diff
@@ -27,7 +27,7 @@ blue {
   #    then receives an information email with summary of registration
   #   `no-email` the user chooses his password upon registration and receives no
   #   email
-  registration-confirmation = "email-confirmation"
+  registration-confirmation = "no-email"
```

Comment:
You must fix the test case configuration so that a confirmation email is sent to verify the process. (see https://github.com/gnieh/bluelatex/blob/master/src/main/configuration/org.gnieh.blue.common/application.conf)

Moreover I would rather use `email-summary` as a default, which is the behavior of several services when ou register.